src/nl/hannahsten/texifyidea/action/wizard/graphic/InsertGraphicWizardDialogWrapper.kt

```diff
@@ -121,7 +122,7 @@ open class InsertGraphicWizardDialogWrapper(val initialFilePath: String = "") :
     /**
      * Contains the label for the figure.
      */
-    private val txtLabel = JBTextField("fig:")
+    private val txtLabel = JBTextField("fig:" + File(initialFilePath).nameWithoutExtension)
```

Comment:
The method `String.formatAsLabel()` would be useful here

Reply:
I'm pretty sure I just copy pasted that from somewhere :eyes:

Reply:
Sure, but if all of TeXiFy was perfect already, you wouldn't have to fix so many bugs 😄